--- a/net/minecraft/client/gui/screens/worldselection/WorldSelectionList.java
+++ b/net/minecraft/client/gui/screens/worldselection/WorldSelectionList.java
@@ -361,6 +_,7 @@
          RenderSystem.enableBlend();
          p_281612_.m_280163_(this.f_101698_.m_289196_(), p_282820_, p_283181_, 0.0F, 0.0F, 32, 32, 32, 32);
          RenderSystem.disableBlend();
+         renderExperimentalWarning(p_281612_, p_283204_, p_283025_, p_283181_, p_282820_);
          if (this.f_101693_.f_91066_.m_231828_().m_231551_() || p_283396_) {
             p_281612_.m_280509_(p_282820_, p_283181_, p_282820_ + 32, p_283181_ + 32, -1601138544);
             int j = p_283204_ - p_282820_;
